lib/mix/tasks/spark.formatter.ex

```diff
@@ -10,7 +10,11 @@ if Code.ensure_loaded?(Sourceror) do
 
     @spec run(term) :: no_return
     def run(opts) do
-      Mix.Task.run("compile")
+      # Compile if safe to do so (not at umbrella root)
+      unless Mix.Project.umbrella?() do
```

Comment:
```suggestion
      if not Mix.Project.umbrella?() do
```

`unless` is deprecated.